Dependency on python-arango >= 8.1.7 should be enforced

It seems that arango-rdf 1.2.2 needs python-arango>=8.1.7, otherwise, when testing the example, I get the following exception:

`>>> adbrdf.rdf_to_arangodb_by_rpt(name="BeatlesRPT", rdf_graph=beatles(), overwrite_graph=True)
(RDF → ADB): Flatten Reified Triples ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% (0/0) 0:00:00
(RDF → ADB): RPT ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% (0/28) 0:00:00[2025/07/22 10:41:19 +0200] [44289] [ERROR] - arango_rdf: Error inserting documents: Collection.insert_many() got an unexpected keyword argument 'raise_on_document_error'
(RDF → ADB): RPT ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% (28/28) 0:00:00
     (RDF → ADB): Import 'BeatlesRPT_URIRef' (13) ▰▱▱▱▱▱▱ 0:00:00
Traceback (most recent call last):
  File "/Users/regisarchambault/Documents/temp/wheel/.venv/lib/python3.13/site-packages/arango_rdf/main.py", line 3554, in __insert_adb_docs
    result = self.db.collection(col).insert_many(
        doc_list, **adb_import_kwargs
    )
TypeError: Collection.insert_many() got an unexpected keyword argument 'raise_on_document_error'`

As a matter of fact, the 'raise_on_document_error' argument was added to the insert_many() method in python-arango 8.1.7

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -32,7 +32,7 @@ classifiers = [
 
 dependencies = [
     "rdflib>=6.0.0",
-    "python-arango~=8.0",
+    "python-arango>=8.1.7,<9",
     "cityhash>=0.4.6",
     "requests>=2.27.1",
     "rich>=12.5.1",

